src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -97,13 +98,18 @@ public void setPersons(List<Person> persons) {
         internalList.setAll(persons);
     }
 
+    public void sortPersons(Comparator<Person> c) {
+        internalList.sort(c);
+    }
```

Comment:
Should probably remove this as it is only used by the unused sortPersons method in AddressBook